GOVERNANCE.md

```diff
@@ -0,0 +1,126 @@
+Governance for the R data.table project
+
+# Purpose and scope
+
+## This document
+
+The purpose of this document is to define how people related to the project work together, so that the project can expand to handle a larger and more diverse group of contributors.
+
+## The R package
+
+The purpose of the project is to maintain the R data.table package, which is guided by the following principles:
+
+* No external Imports/LinkingTo/Depends dependencies (external meaning those not maintained by the project)
+* Few (if any) Suggests/Enhances dependencies
+* Time & memory efficiency
+* Concise syntax (minimal redundancy in code)
+* Stable code base (strong preference for user-friendly back-compatibility with data.table itself and with old versions of R)
+* Comprehensive and accessible documentation and run-time signals (errors, warnings)
+* Clear error and warning messages
+
+To prioritize developer time, we define what is in and out of current scope. Feature requests in issues and pull requests that are out of current scope should be closed immediately, because they are not the current priority. If someone wants to contribute code that is currently out of scope, they first have to make a pull request that changes the scope as defined below.
+
+The current scope of package functionality includes:
+* Data manipulation and analysis 
+    * reshaping/pivoting
+    * aggregation/summarizing
+    * subset/bind rows (rbindlist)
```

Comment:
rbindlist doesn't really fit in this point. It is a one of the ?setops so if we want to have it here then we can add point about setops